overlord/devicestate/devicemgr.go

```diff
@@ -670,7 +670,11 @@ func (m *DeviceManager) Ensure() error {
 	var errs []error
 
 	if err := m.ensureSeeded(); err != nil {
-		errs = append(errs, err)
+		// XXX: avoid duplicated warnings
```

Comment:
Right, we need to do something to prevent the warning every few minutes. Perhaps a bool flag on the manager would suffice.

Reply:
Yeah, it's a bit of a question - I was thinking we could keep it this way because the problem stay. But if we do I think the warning needs to have a hint how to resolve the issue. So maybe a link to a forum topic about this? Something like:
```
Cannot seed the system: %v

This usually indicates an error in packaging of snapd. Please contact your distribution about this error. To fix this yourself see https://forrum.snapcraft.io/t/1234
```
And in that forum topic we could describe that "rm -f /var/lib/snapd/seed/seed.yaml" is probably the best one can do in this situation.

We still need to do something independent of this for the people on early focal with the issue, fortunately zyga had a system with the broken seed so we can use that to investigate and use the systems from Mark and Ryan as testbeds for the fix.

Reply:
I will remove the XXX in a followup, don't want to re-run spread just for this :(